pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 120
```

Comment:
nice!

Reply:
Very nice

Reply:
Maybe we can have the github action use this file?